README.md

```diff
@@ -2,153 +2,270 @@
 
 Java library to access the Authy API
 
+## Dependencies
+
+### JSON Library
+This project use [json.org](https://github.com/douglascrockford/JSON-java) you can find
+the [json.org jar versions here](https://search.maven.org/#search|gav|1|g%3A%22org.json%22%20AND%20a%3A%22json%22)
+
+* Using Ant you don't need to include json.org since ant is including it in the final jar.
+
+* Using Maven you need to include the [json.org jar](https://search.maven.org/#search|gav|1|g%3A%22org.json%22%20AND%20a%3A%22json%22) in your jar external libraries.
+
 ## Compilation
 
+
 ### Using Apache Ant
 
 Use `ant` to generate `authy-java.jar` or use the jar file in `dist` directory.
 
-	ant compress
-	
+ant compress
+
 This will generate a jar file in `dist` folder.
 
+If you want to use ant step by step you can follow these commands:
+```shell
+$ ant clean
+$ ant compile
+$ ant compress
+```
+or in one line
+
+```shell
+$ ant clean ; ant compile ; ant compress
+```
+
```

Comment:
I think it's not necessary to put the one line code. Actually, I would let only the `ant compress` code.